src/OSPSuite.UI/Views/Importer/ColumnMappingView.cs

```diff
@@ -58,13 +63,21 @@ public ColumnMappingView(IImageListRetriever imageListRetriever)
          var unitInformationTip = new SuperToolTip();
          unitInformationTip.Items.Add(Captions.Importer.UnitInformationDescription);
 
-         _repositoryItemPopupContainerEdit.Buttons[0].Kind = ButtonPredefines.Combo;
-         _repositoryItemPopupContainerEdit.PopupControl = _popupControl;
-         _repositoryItemPopupContainerEdit.CloseOnOuterMouseClick = false;
-         _repositoryItemPopupContainerEdit.QueryDisplayText += (o, e) => queryDisplayText(e);
-         _repositoryItemPopupContainerEdit.CloseUp += (o, e) => closeUp(e);
-         _repositoryItemPopupContainerEdit.CloseUpKey = new KeyShortcut(Keys.Enter);
-         _repositoryItemPopupContainerEdit.AllowDropDownWhenReadOnly = DefaultBoolean.True;
+         _repositoryMappingPopupContainerEdit.Buttons[0].Kind = ButtonPredefines.Combo;
+         _repositoryMappingPopupContainerEdit.PopupControl = _mappingPopupControl;
+         _repositoryMappingPopupContainerEdit.CloseOnOuterMouseClick = false;
+         _repositoryMappingPopupContainerEdit.QueryDisplayText += (o, e) => queryDisplayText(e);
+         _repositoryMappingPopupContainerEdit.CloseUp += (o, e) => closeUpMapping(e);
+         _repositoryMappingPopupContainerEdit.CloseUpKey = new KeyShortcut(Keys.Enter);
+         _repositoryMappingPopupContainerEdit.AllowDropDownWhenReadOnly = DefaultBoolean.True;
+
+         _repositoryMetaDataPopupContainerEdit.Buttons[0].Kind = ButtonPredefines.Combo;
```

Comment:
maybe create a method to avoid code duplication (same code exactly) just different control

Reply:
packed it in private initializeButton()